src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private StringBuilder totalSalaryInfo = new StringBuilder();
```

Comment:
make this variable local

Reply:
Please discribe, when we make local variables and when class variables.
Dasha sad, "you have to make constructors in class body, because object create once."
But your propose is vise versa.

Reply:
if you have an opportunity not to make variable global then it's better to do this, in this case, you don't need to make this field global, it's enough to make it local because you use it only in one method